CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Tarmo Pikaro
+# Distributed under the Boost Software License, Version 1.0.
+# See accompanying file LICENSE_1_0.txt or copy at https://www.boost.org/LICENSE_1_0.txt
+
+cmake_minimum_required( VERSION 3.5 )
+project( BoostDateTime LANGUAGES CXX)
+
+add_library( boost_date_time
+    src/gregorian/greg_month.cpp
+)
+add_library( Boost::chrono ALIAS boost_date_time )
+
+target_include_directories( boost_date_time PUBLIC include )
+
+target_link_libraries( boost_date_time
+    PUBLIC
+        Boost::config
```

Comment:
my 'cmake foo' is used to older versions.  Does the 'PUBLIC' mean header only bc there's no library to link with boost::config.  And what about lexical_cast, shared_ptr, and other libraries?  serialization *is* needed to actually link if and only if you use the serialization code.

Reply:
At the moment I'm not so sure about my cmake commits in overall. My current target is just to make boost / stacktrace working, but I have also slightly bigger questions in overall concerning boost build / cmake build system - I'm thinking of driving my changes in two directions - boost stacktrace would have standalone cmake support (independent from boost cmake), also I'm thinking of completely replacing boost build with cmake, but thinking is because I did not manage to get in touch with either boost build or boost cmake mail lists, suspect they are dead / not maintained.

Threat my commits to date_time to cmake just an early prototype of cmake for date_time.

Reply:
Ok -- could we get a comment to that effect in the file?  I don't want people to get the idea that this is 'official' or something.  And I thought @pdimov was probably leading the cmake charge?